www/src/util/serverResponses.d.ts

```diff
@@ -12,4 +12,6 @@ export type User = {
     updated_at: db.TimestampTzString;
```

Comment:
Can you help me understand what is going on in this file. Why do we need to wrap this type?

Reply:
The server response JSON looks like 
```json
{
  "user" : { ... }
}
```
according to [`getMe.ts`](https://github.com/UAlbertaCompEClub/compe-plus/blob/35bbc9d6d2fd28bf45a0beae3316d28d0ec29a84/api/src/controllers/user/getMe.ts#L7-L9).

It still works fine without since we're just checking if the response returns any data at all but might be better to actually type what the server actually responds with